src/app/crud/base.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, Dict, Generic, List, Optional, Type, TypeVar, Union
+
+from fastapi.encoders import jsonable_encoder
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.db.base_class import Base
+from sqlalchemy.sql.expression import select
+
+ModelType = TypeVar("ModelType", bound=Base)
```

Comment:
For the sake of not indicating a closer grouping of imports with this line than with the top level class definition here we should have two newlines between imports and this statement